### ISSUE TITLE
Need full integration for Orders and Customer with Wickedreports

### ISSUE THREAD

**Issue by christophcemper** [Simplero/Roadmap#134]
We need the full integration with Wickedreports from Simplero sales and leads.

What we would need is pushing Contact Data and Order Data (initial and recurring payments) 
with their spec here https://wickedreports.helpdocs.com/getting-started-with-wicked-reports/wicked-reports-api-documentation

WickedReports is very interested in making this happen as well,
just reach out to them. Happy to share costs

This would give you a lot of exposure to top marketers 
as Frank Kern and others are currently pushing WickedReports *a lot*

**Comment by nicolabird** [comment]
YES YES YES! It's the only thing that ever tempts me away from simplero - I'd really like to be able to  use wicked reports. They are up for doing an integration but just for me that would be a prohibitive cost - they might be more interested if more of us were keen.

**Comment by christophcemper** [comment]
Any update on this @nickbruun ? think this would really help a lot

**Comment by nickbruun** [comment]
@christophcemper nope. I can't promise when we'd get around to taking a look at these types of integrations, but looking at the roadmap it'd not be until next year. Our focus, relating to other similar requests, would potentially also be on providing better analytical tools in Simplero first, but in all honesty we have not started planning any of this in detail yet :)
